ReplyAdaptabilityCodelab/app/src/main/java/com/example/reply/data/EmailsRepository.kt

```diff
@@ -0,0 +1,13 @@
+package com.example.reply.data
+
+import kotlinx.coroutines.flow.Flow
+
+/**
+ * An Interface contract to get all enails info for a User.
+ */
+interface EmailsRepository {
```

Comment:
why do we need an interface? Can we just directly use the implementation?

Reply:
We can skip the interfaces, I was just following the contract structure of Interface + Implementation for this sample.